src/Ormolu/Printer/Combinators.hs

```diff
@@ -270,3 +277,28 @@ comma = txt ","
 
 space :: R ()
 space = txt " "
+
+----------------------------------------------------------------------------
+-- Layout
+
+data LayoutOptions
```

Comment:
Documentation is very important, if you look at other functions in this module, they are all documented very carefully because this module is important for building everything else. Good documentation also makes it easier for reviewers to understand how things work.

I like how you describe your PRs, surely the actual code which will be read perhaps many years later and not once deserves not less attention :)